common/src/java/org/apache/hive/http/JMXJsonServlet.java

```diff
@@ -241,40 +241,34 @@ private void listBeans(JsonGenerator jg, ObjectName qry, String attribute,
         } catch (AttributeNotFoundException e) {
           // If the modelerType attribute was not found, the class name is used
           // instead.
-          LOG.error("getting attribute " + prs + " of " + oname
-              + " threw an exception", e);
+          LOG.error("getting attribute {} of {} threw an exception", prs, oname, e);
```

Comment:
I am wondering if the exception serialization remains the same after this change since we no longer use the API that accepts a `Throwable`.

Reply:
In addition it is strange that we use **two** placeholders `{}` but we pass **three** arguments.